CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
```

Comment:
Do you want to start with a 1.0 release? I suggest we start with a 0.1.0 release and release 1.0 once we are sure we have preserved existing functionality.

Reply:
all exisiting functionality from `utils-provider` is there, including all the examples and tests

Reply:
but if you want to look at the new Provider Framework, it's a completely diff story

Reply:
but let's start with `0.1.0` in any case, updated